onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2964,7 +2964,96 @@ It's an extension of Gelu. It takes the sum of input A and bias input B as the i
           ctx.getOutputType(0)
               ->CopyFrom(input_type->optional_type().elem_type());
           });
-  
+
+  static const char* GridSampler_ver1_doc = R"DOC(
+      Given an `input` and a flow-field `grid`, computes the `output` using `input` values and pixel locations from `grid`.
+      Currently, only spatial (4-D) inputs are supported. For `input` with shap (N, C, H, W) and `grid` with shape (N, H_out, W_out, 2),
+      the `output` will have shape (N, C, H_out, W_out).
+      For each output location `output[n, :, h, w]`, the size-2 vector `grid[n, h, w]` specifies `input` pixel locations `x` and `y`,
+      which are used to interpolate the output value `output[n, :, h, w]`.
+      The GridSampler operator is often used in conjunction with affine_grid doing grid generator
```

Comment:
there is no affine_grid operator in ONNX or ORT contrib ops... do we need to add this op too?

Reply:
The affine_grid is used to create the `Grid` input of grid_sample.
The github community and fomm model only request for `grid_sampler`, so I think we don't need this op now.